### PR TITLE
login, logout and register

### PR DESCRIPTION
**Pull request by gavin-go-dang**
-Register new user
-Login
-Logout

### CODE REVIEW

File: mysite/mysite/settings/settings_local.py [hunk 1]
```diff
@@ -1,4 +1,2 @@
 from .base import *
-
-DEBUG = True
-# INSTALLED_APPS += 
\ No newline at end of file
+DEBUG=True
```
Comment (thengoc-bui-goldenowl) [inline mysite/mysite/settings/settings_local.py]: code style

File: mysite/account/views.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                user.save()
+                return redirect('login')
+        else:
+            messages.info(request, 'Both passwords are not matching')
+            return redirect('register')
+    else:
+        return render(request, 'registeration.html')
+
+
+def login(request):
```
Comment (thengoc-bui-goldenowl) [inline mysite/account/views.py]: how about if the user has logged in ? Try researching class base-view, and apply

File: mysite/account/views.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+from django.contrib.auth.models import User, auth
+# Create your views here.
+
+
+
+def home(request):
+    return render(request, 'home.html')
+
+
+def register(request):
```
Comment (thengoc-bui-goldenowl) [inline mysite/account/views.py]: Try using class based-view

File: mysite/account/templates/home.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```
Comment (thengoc-bui-goldenowl) [inline mysite/account/templates/home.html]: Let's base template and extend.
keywords 
`extend
include
`

File: mysite/account/templates/home.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <title>Test page</title>
+</head>
+
+<body>
+
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
```
Comment (thengoc-bui-goldenowl) [inline mysite/account/templates/home.html]: can put in head tag

**Review by thengoc-bui-goldenowl — CHANGES_REQUESTED** [review]

File: mysite/account/templates/header.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (thengoc-bui-goldenowl) [inline mysite/account/templates/header.html]: This template includes not only header but also others, the name shouldn't be `header.html`, the name likes `base.html`, ...

File: mysite/account/views.py [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+from django.contrib import messages
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User, auth
+from django.views import View
+from django.views.generic import View
+from django.contrib.auth.views import LoginView, LogoutView
+# Create your views here.
+
+
+class HomeView(View):
```
Comment (thengoc-bui-goldenowl) [inline mysite/account/views.py]: can use TemplateView

**Review by thengoc-bui-goldenowl — CHANGES_REQUESTED** [review]

**Comment by gavin-go-dang** [comment]
Login screen:
image
Wrong password or username:
image
Home screen(Not logging):
image
Home screen(logged):
image
Register:
image
Wrong information register:
image

**Pull request merged**